v2/internal/attacktechniques/aws/lateral-movement/ssm-start-session/main.go

```diff
@@ -0,0 +1,81 @@
+package aws
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
+	"strings"
+)
+
+//go:embed main.tf
+var tf []byte
+
+func init() {
+	const codeBlock = "```"
+	stratus.GetRegistry().RegisterAttackTechnique(&stratus.AttackTechnique{
+		ID:           "aws.lateral-movement.ssm-start-session",
```

Comment:
I would tend to classify it as execution, wdyt? we have `VM run command`  in execution for Azure

Reply:
That will probably depend on the attack itself and what is the objective of the attacker. But, the way I see it here is that the end goal is not to execute code in the EC2 but to move to a different environment.  I think it is also well-aligned with 
Remote Services: Direct Cloud VM Connections - ID: T1021.008

Probably for SSM SendCommand execution makes more sense as the goal might be the execution of the command not to move to a different environment.

Reply:
alright, no strong opinion besides that it should be consistent across clouds. I think I will still move it to `execution` to be consistent with https://attack.mitre.org/techniques/T1651/

Reply:
done in [928ceb2](https://github.com/DataDog/stratus-red-team/pull/477/commits/928ceb2b39bd9cab777c85c78b94de6891e29b45)